src/app/features/shared/components/molecules/FlashMessages/FlashMessages.tsx

```diff
@@ -2,17 +2,21 @@ import React from "react"
 import { useLocation } from "@reach/router"
 
 import { useFlashMessages } from "app/state/flashMessages/useFlashMessages"
-// TODO: Do not import this from amsterdam-react-final-form better from ASC
-import Alert from "amsterdam-react-final-form/components/unbound/Alert"
+import { Alert, themeSpacing } from "@amsterdam/asc-ui"
+import styled from "styled-components"
+
+const StyledAlert = styled(Alert)`
+  margin-bottom: ${ themeSpacing(5) }
```

Comment:
Not better to wrap the Alert in a `<Row/>`?

Reply:
You are right, I changed it